annotation-processor/src/main/kotlin/pl/touk/krush/source/TablesGenerator.kt

```diff
@@ -262,6 +253,13 @@ class TablesGenerator : SourceGenerator {
         return codeBlockBuilder.build()
     }
 
+
```

Comment:
\[ktlint\] WARNING: \[no\-consecutive\-blank\-lines\] Needless blank line\(s\) in column 1